.circleci/images/primary/Dockerfile-2.4.10

```diff
@@ -1,7 +1,8 @@
-# https://hub.docker.com/layers/ruby/library/ruby/2.4.10-stretch/images/sha256-e932ae985bce29c1e1b845332af331fbc7b2cc9a7cb5477b1727f665c78aba0a
+# This image supports multiple platforms, see README for build instructions
```

Comment:
Minor: Perhaps worth adding the same comment on all images? (Or remove from all -- now that all have been fixed)

Reply:
Removing sounds good!